subprojects/tooling-api/src/main/java/org/gradle/tooling/internal/protocol/events/InternalTaskResult.java

```diff
@@ -22,4 +22,8 @@
  * @since 2.5
  */
 public interface InternalTaskResult extends InternalOperationResult {
+    /**
+     * @since 5.1
+     */
+    String getOutcomeDescription();
```

Comment:
Why was this added?

Reply:
Because `DefaultTaskFinishedProgressEvent` now receives an `InternalTaskResult` instead of an `AbstractTaskResult` and needs to access it in its `getDisplayName()` method.